<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSetting_Dialog</class>
 <widget class="QDialog" name="CameraSetting_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.943, y1:0.193182, x2:0.187, y2:0.818182, stop:0 rgba(85, 255, 255, 255), stop:0.982955 rgba(255, 213, 162, 255));
font: 14pt &quot;华文新魏&quot;;</string>
  </property>
  <widget class="QGroupBox" name="Left_CameraGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>左摄像头</string>
   </property>
   <widget class="QComboBox" name="LeftCamera_DeviceList">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="left_camera_deviceName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.943, y1:0.193182, x2:0.187, y2:0.818182, stop:0 rgba(85, 255, 255, 0), stop:0.982955 rgba(255, 213, 162, 0));</string>
    </property>
    <property name="text">
     <string>设备名：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="leftEnable_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.943, y1:0.193182, x2:0.187, y2:0.818182, stop:0 rgba(85, 255, 255, 0), stop:0.982955 rgba(255, 213, 162, 0));</string>
    </property>
    <property name="text">
     <string>使能</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Right_CameraGoup">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>9</y>
     <width>230</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>右摄像头</string>
   </property>
   <widget class="QLabel" name="right_deviceName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.943, y1:0.193182, x2:0.187, y2:0.818182, stop:0 rgba(85, 255, 255, 0), stop:0.982955 rgba(255, 213, 162, 0));</string>
    </property>
    <property name="text">
     <string>设备名：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Right_CameraDeviceList">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rightEnable_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.943, y1:0.193182, x2:0.187, y2:0.818182, stop:0 rgba(85, 255, 255, 0), stop:0.982955 rgba(255, 213, 162, 0));</string>
    </property>
    <property name="text">
     <string>使能</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="SettingBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确认设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="signleCameraset">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>461</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>单目设置</string>
   </property>
   <widget class="QPushButton" name="CalibrationSingleCamera">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>相机标定</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TwiceCameraset">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>461</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>双目设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
